scripts/project_completion.py

```diff
@@ -0,0 +1,166 @@
+#!/bin/env python
+"""Print the analysis status for samples, libpreps, and seqruns within a project."""
+
+from __future__ import print_function
+
+import argparse
+import collections
+import functools
+import os
+import sys
+import tempfile
+import time
+
+from ngi_pipeline.conductor.flowcell import organize_projects_from_flowcell
+from ngi_pipeline.database.classes import CharonSession, CharonError
+from ngi_pipeline.engines.piper_ngi.local_process_tracking import update_charon_with_local_jobs_status
+from ngi_pipeline.utils.classes import with_ngi_config
+from ngi_pipeline.utils.filesystem import locate_flowcell, locate_project
+
+print_stderr = functools.partial(print, file=sys.stderr)
+
+def project_summarize(projects, brief=False, verbose=False):
+    update_charon_with_local_jobs_status(quiet=True) # Don't send mails
+    charon_session = CharonSession()
+    projects_list = []
+    for project in projects:
+        print_stderr('Gathering information for project "{}"...'.format(project))
+        project_dict = {}
+        try:
+            project = charon_session.project_get(project)
+        except CharonError as e:
+            print_stderr('Project "{}" not found in Charon; skipping ({})'.format(project, e), file=sys.stderr)
+            continue
+        project_dict['name'] = project['name']
+        project_dict['id'] = project['projectid']
+        project_dict['status'] = project['status']
+        samples_list = project_dict['samples'] = []
```

Comment:
Yeah good question actually... I don't know, I think in this case I'd like it to fail because it would imply that Charon has changed upstream and we need to update this script, and as it's not mission-critical I'd rather someone complained than the script just broke and we didn't know why